filetests/isa/intel/binary64.cton

```diff
@@ -145,6 +145,149 @@ ebb0:
+    [-,%rdx]            v417 = uload8.i64 v2-50         ; bin: 48 0f b6 56 ce
+    ; asm: movsbq 50(%rcx), %rdi
+    [-,%rdi]            v418 = sload8.i64 v1+50         ; bin: 48 0f be 79 32
+    ; asm: movsbq -50(%rsi), %rdx
+    [-,%rdx]            v419 = sload8.i64 v2-50         ; bin: 48 0f be 56 ce
+
+    ; Register-indirect with 32-bit signed displacement.
+
+    ; asm: movq %rcx, 10000(%rsi)
+    store v1, v2+10000                          ; bin: 48 89 8e 00002710
+    ; asm: movq %rsi, -10000(%rcx)
+    store v2, v1-10000                          ; bin: 48 89 b1 ffffd8f0
+    ; asm: movl %ecx, 10000(%rsi)
+    istore32 v1, v2+10000                       ; bin: 40 89 8e 00002710
+    ; asm: movl %esi, -10000(%rcx)
+    istore32 v2, v1-10000                       ; bin: 40 89 b1 ffffd8f0
+    ; asm: movw %cx, 10000(%rsi)
+    istore16 v1, v2+10000                       ; bin: 66 40 89 8e 00002710
+    ; asm: movw %si, -10000(%rcx)
+    istore16 v2, v1-10000                       ; bin: 66 40 89 b1 ffffd8f0
+    ; asm: movb %cl, 10000(%rsi)
+    istore8 v1, v2+10000                        ; bin: 40 88 8e 00002710
+    ; asm: movb %sil, 10000(%rcx)
+    istore8 v2, v1+10000                        ; bin: 40 88 b1 00002710
+
+    ; asm: movq 50000(%rcx), %rdi
+    [-,%rdi]            v480 = load.i64 v1+50000           ; bin: 48 8b b9 0000c350
+    ; asm: movq -50000(%rsi), %rdx
+    [-,%rdx]            v481 = load.i64 v2-50000           ; bin: 48 8b 96 ffff3cb0
+    ; asm: movl 50000(%rcx), %edi
+    [-,%rdi]            v482 = uload32.i64 v1+50000        ; bin: 40 8b b9 0000c350
+    ; asm: movl -50000(%rsi), %edx
+    [-,%rdx]            v483 = uload32.i64 v2-50000        ; bin: 40 8b 96 ffff3cb0
+    ; asm: movslq 50000(%rcx), %rdi
+    [-,%rdi]            v420 = sload32.i64 v1+50000        ; bin: 48 63 b9 0000c350
+    ; asm: movslq -50000(%rsi), %rdx
+    [-,%rdx]            v421 = sload32.i64 v2-50000        ; bin: 48 63 96 ffff3cb0
+    ; asm: movzwq 50000(%rcx), %rdi
+    [-,%rdi]            v422 = uload16.i64 v1+50000        ; bin: 48 0f b7 b9 0000c350
+    ; asm: movzwq -50000(%rsi), %rdx
+    [-,%rdx]            v423 = uload16.i64 v2-50000        ; bin: 48 0f b7 96 ffff3cb0
+    ; asm: movswq 50000(%rcx), %rdi
+    [-,%rdi]            v424 = sload16.i64 v1+50000        ; bin: 48 0f bf b9 0000c350
+    ; asm: movswq -50000(%rsi), %rdx
+    [-,%rdx]            v425 = sload16.i64 v2-50000        ; bin: 48 0f bf 96 ffff3cb0
+    ; asm: movzbq 50000(%rcx), %rdi
+    [-,%rdi]            v426 = uload8.i64 v1+50000         ; bin: 48 0f b6 b9 0000c350
+    ; asm: movzbq -50000(%rsi), %rdx
+    [-,%rdx]            v427 = uload8.i64 v2-50000         ; bin: 48 0f b6 96 ffff3cb0
+    ; asm: movsbq 50000(%rcx), %rdi
+    [-,%rdi]            v428 = sload8.i64 v1+50000         ; bin: 48 0f be b9 0000c350
+    ; asm: movsbq -50000(%rsi), %rdx
+    [-,%rdx]            v429 = sload8.i64 v2-50000         ; bin: 48 0f be 96 ffff3cb0
```

Comment:
This is great, but please add similar tests for `*.i32` instructions to the `%I32` function below.